code/modules/crafting/recipes.dm

```diff
@@ -669,3 +669,11 @@
 	tools = list(TOOL_SCREWDRIVER, TOOL_WIRECUTTER)
 	reqs = list(/obj/item/clothing/glasses/sunglasses/reagent = 1)
 	category = CAT_CLOTHING
+
+/datum/crafting_recipe/ghostsheet
+	name = "Ghost Sheet"
+	result = /obj/item/clothing/suit/ghost_sheet
+	time = 5
+	tools = list(TOOL_WIRECUTTER)
+	reqs = list(/obj/item/bedsheet = 1)
```

Comment:
This allows you to use any kind of bedsheet, I was intending for it to only use the plain white kind, and I don't know how to restrict it to that type. It realistically doesn't matter, but if I knew how I'd change it.